app/models/ansible_variable.rb

```diff
@@ -12,8 +12,8 @@ def ansible?
     true
   end
 
-  def self.humanize_class_name
-    'Ansible variable'
+  def self.humanize_class_name(name = 'Ansible variable')
```

Comment:
where do we use this new feature of this method? IIRC this is some rails class which expects 0 arguments. We override it in core for variable lookup key but accepting options. If we need this, should we follow the same pattern? But still, I don't see this parameter being used anywhere, so first I'm trying to understand the use of it.

Reply:
I was getting wrong number of arguments, but I can no longer reproduce, so I reverted to the original state.